src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import { CSSProperties } from "react";
+import Image from "next/image";
+import userProfile from "@/assets/userProfile.svg";
+import * as S from "./Avatar.styles";
+
+interface AvatarProps {
+  size?: "xs" | "sm" | "md" | "lg";
+  name?: string;
+  src?: string;
+  style?: CSSProperties;

```

Comment:
[P5]

정말 그럴일이 많지 않을것같은데 호오오옥시 추후를 위해 아래 prop 한개 추가하는건어떨까요!?
```suggestion
  style?: CSSProperties;
  imageStyle?: CSSProperties;
```